src/System.Net.Sockets/src/System.Net.Sockets.csproj

```diff
@@ -1,4 +1,5 @@
-﻿<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
This changed quite a bit. Please change only one line to avoid complicated merges (the single line referencing the new implementation file).